src/_companies/europe/sweden/gothenburg/varvet/index.md

```diff
@@ -0,0 +1,23 @@
+---
+name:  "Varvet"
+
+continent: "Europe"
+country: "Sweden"
+region: "Gothenburg"
+city: "Gothenburg"
+
+website: https://www.varvet.com/
+created_at: "2020-02-18 18:20:02.02"
```

Comment:
```suggestion
created_at: 2020-02-18 18:20:02.02
```
Without the quotes, ruby will actually parses this from YAML as a Time instance. That should work better for the sorting of the newest companies, since all other companies also don't use a string here :)

Reply:
Gotcha, done!